qutip/core/cy/qobjevo.pyx

```diff
@@ -251,10 +251,10 @@ cdef class QobjEvo:
             out = _ConstantElement(op.copy() if copy else op)
             qobj = op
         elif isinstance(op, list):
+            order = 0 if use_step_func else 3
```

Comment:
@BoxiLi, we are changing the option to use step interpolation in `QobjEvo` and `coefficient` from `step_interpolation` or `_stepInterpolation` to `order=0` in both place. It will probably break some of `qutip-qip`, please tell us if now is a bad time to change that.